lib/native/auth.ts

```diff
@@ -189,9 +177,6 @@ export interface AwsSigningConfig {
 
     /** Controls what header, if any, should be added to the request, containing the body value */
     signed_body_header?: AwsSignedBodyHeaderType;
-
-    /** Query param signing only: how long the pre-signed URL is valid for */
-    expiration_in_seconds?: number;
```

Comment:
Was this intentional?

Reply:
That's a different expiration time.  This is something that gets written into the sigv4 signing process and has nothing to do with AWS Credentials expiration.